app/build.gradle

```
@@ -1,7 +1,10 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
+    id 'kotlin-kapt'
 }
+apply plugin: 'dagger.hilt.android.plugin'
```

```suggestion
plugins {
    id 'com.android.application'
    id 'kotlin-android'
    id 'kotlin-kapt'
    id 'dagger.hilt.android.plugin'
}
```
이렇게 해보실래요?? 아마 구 버전 플로그인 문법이랑 요즘 플로그인 선언 문법이랑 동시에 존재해서 충돌난것 같아요!

---

마찬가지 입니다 ! 원래 이렇게 했으나 에러가 나서 그냥 공식문서와 토시 하나 틀리지 않게 했음을 보여드리려고 이렇게 커밋했네요..